smurfs/src/components/Smurfs.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component, Fragment } from "react";
+import { connect } from "react-redux";
+import { getSmurfs, addSmurf } from "../actions";
+
+class Smurfs extends Component {
+  constructor() {
+    super();
+    this.state = {
+      smurfBio: {
+        name: "",
+        age: "",
+        height: ""
+      }
+    };
+  }
+
+  componentDidMount() {
+    this.props.getSmurfs();
+  }
+
+  handleChanges = e => {
+    this.setState({
+      smurfBio: {
+        ...this.state.smurfBio,
+        [e.target.name]: e.target.value
+      }
+    });
+  };
+
+  addSmurf = e => {
+    console.log(this.state);
+    e.preventDefault();
+    this.props.addSmurf(this.state.smurfBio);
+    this.setState({
+      smurfBio: {
+        name: "",
+        age: "",
+        height: ""
+      }
+    });
+    console.log("addbutton");
+  };
+
+  render() {
+    console.log(this.props.smurfBio);
+    return (
+      <Fragment>
+        <div className="ui compact message">
+          <div className="content">
+            <p>
+              Want to become a part of the <strong>Smurf Family?</strong> add
+              yourself below.
+            </p>
+          </div>
+        </div>
+        {this.props.smurfBio.map(smurf => (

```

Comment:
The naming convention here is very confusing. I would rename this to `smurfs` or something similar to differentiate from smurfsBio on state.